src/HeaderBar.js

```diff
@@ -19,8 +19,9 @@ function HeaderBar(props) {
 }
 
 function HeaderIcon(props) {
+  const publicUrl = process.env.PUBLIC_URL ? process.env.PUBLIC_URL : '/';
```

Comment:
If you decide to change to NODE_ENV === 'development', then you should also change the check here

Reply:
So it would become `process.env.NODE_ENV === 'development' ? process.env.PUBLIC_URL : '/'` ? That seems unnecessarily indirect.